alacritty_terminal/src/term/mod.rs

```diff
@@ -1785,6 +1785,10 @@ impl<T: EventListener> ansi::Handler for Term<T> {
                 if self.mode.contains(TermMode::ALT_SCREEN) {
                     self.grid.region_mut(..).each(|c| c.reset(&template));
                 } else {
+                    let template = Cell {
+                        c: template.c,
```

Comment:
Do we need a background color?

Reply:
Oh, I completely forgot that. Of course we do. Good catch, thanks.